CONTRIBUTING.md

```diff
@@ -243,7 +243,7 @@ Get-Process | Where-Object { $_.ProcessName -like "*azmcp*" } | Select-Object Id
 
 To ensure consistent code quality, code format checks will run during all PR and CI builds.
 
-To catch format errors early, run `dotnet format src/AzureMcp.sln` before submitting.
+To catch format errors early, run `dotnet format AzureMcp.sln` before submitting.
```

Comment:
You can just run `dotnet format`

Reply:
Are you suggesting changing this command to just `dotnet format`?

Reply:
Yes, please while you are in there.